File: week-02/day-4/elementfinder.py
# Check if the list contains "7" if it contains print "Hoorray" otherwise print "Noooooo"

numbers = [1, 2, 3, 4, 5, 6, 8];

def in_list():
    a = "Noooooo!"
    for i in numbers:
        if i == 7:
            a = "Hoooooray!"

    return a

print(in_list())

File: week-07/day-3/elementfinder.js
'use strict';

// Check if the array contains "7" if it contains print "Hoorray" otherwise print "Noooooo"

var numbers = [1,2,3,4,5,6,7,8];

function isInList(list) {
    if (list.includes(7)) {
        return 'Hoorray';
    } else {
        return 'Noooooo';
    }
}

console.log(isInList(numbers));
